pkg/capabilities/triggers/streams/streams_trigger-schema.json

```diff
@@ -0,0 +1,92 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://github.com/smartcontractkit/chainlink-common/pkg/capabilities/triggers/streams/streams-trigger@1.0.0",
+  "description": "Streams Trigger",
+  "$defs": {
+    "feedId": {
+      "type": "string",
+      "pattern": "^0x[0-9a-f]{64}$",
+      "description": "The ID of the data feed."
+    },
+    "Feed": {
+      "type": "object",
+      "properties": {
+        "feedId": {
+          "$ref": "#/$defs/feedId"
+        },
+        "fullReport": {
+          "type": "string",
+          "contentEncoding": "base64",
```

Comment:
Calling out this base64 encoding hack (also on other lines). Not sure what we want to do about it...

Reply:
Do we need to specify contentEncoding at all?

Reply:
It's not really a string, it's bytes under the hood.  Having this would at least allow the engine to potentially do the conversion or something...

We really do need something better long-term, I'm open to a fix now but I don't think it's clear what to do.  Otherwise, maybe some follow-up PR.